### ISSUE TITLE
►►INTERNAL ERROR (UpdtSensorUpdate-'NoneType' object has no attribute 'attributes')

### ISSUE THREAD

**Issue by bryeartem** [gcobb321/icloud3#28]
►►INTERNAL ERROR (UpdtSensorUpdate-'NoneType' object has no attribute 'attributes')

```
'NoneType' object has no attribute 'attributes'
Traceback (most recent call last):
  File "/home/homeassistant/.homeassistant/custom_components/icloud3/device_tracker.py", line 3377, in _update_device_sensors
    badge_attr     = badge_data.attributes
AttributeError: 'NoneType' object has no attribute 'attributes'
```

**Comment by klovatt** [comment]
Seems to be the badge sensor with no valid template.

**Comment by gcobb321** [comment]
Fixed in v1.0.1. This error message was displayed if the 'sensor.devicename_badge' did not exist. A sample sensor setup is in the 'sn_badges.yaml' file in the configuration_files.zip file. Now, it does not exist, one will be created that can be used as the source of your own 'badge'. See the 'Special Sensors' section in the Readme for more information about the '_badge' sensor.
